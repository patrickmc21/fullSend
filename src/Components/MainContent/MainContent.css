.main-content-wrapper {
  padding-top: 50px;
  height: 620px;
  background-color: #f8f7ee;
}

.main-content {
  border: 1px solid #515c28;
  border-radius: 15px;
  margin: auto;
  height: 500px;
  width: 80vw;
  min-width: 300px;
  background-image: url('./images/background.jpg');
  background-size: cover;
  background-repeat: no-repeat;
  box-sizing: border-box;
  box-shadow: 0px 0px 1px 1px #515c28;
}

.main-content-text {
  padding-top: 220px;
  border: 1px solid #727167;
  border-radius: 15px;
  margin: auto;
  height: 100%;
  width: 100%;
  background-color: rgba(0, 0, 0, 0.5);
  box-sizing: border-box;
  box-shadow: 0px 0px 2px 2px #727167;
}

.main-tagline {
  color: #d2cdad;
  font-family: 'Sedgwick Ave Display', cursive;
  font-size: 56px;
  text-align: center;
}

