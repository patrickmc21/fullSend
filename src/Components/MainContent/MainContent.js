import React from 'react';

import './MainContent.css';

const MainContent = () => {
  return (
    <div className='main-content-wrapper'>
      <div className='main-content'>
        <div className='main-content-text'>
          <h1 className='main-tagline'>
            The Rider's Best Friend
          </h1>
        </div>
      </div>
    </div>
  )
}

export default MainContent;