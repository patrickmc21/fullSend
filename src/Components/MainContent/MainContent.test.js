import React from 'react';
import { shallow } from 'enzyme';
import MainContent from './MainContent';

describe('MainContent', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<MainContent />);
    expect(wrapper).toMatchSnapshot();
  });
});