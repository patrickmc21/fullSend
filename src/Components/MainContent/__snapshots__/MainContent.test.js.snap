// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MainContent />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="main-content"
>
        <div
                className="main-content-text"
        >
                <h1
                        className="main-tagline"
                >
                        The Rider's Best Friend
                </h1>
        </div>
</div>,
      "className": "main-content-wrapper",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="main-content-text"
>
          <h1
                    className="main-tagline"
          >
                    The Rider's Best Friend
          </h1>
</div>,
        "className": "main-content",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="main-tagline"
>
            The Rider's Best Friend
</h1>,
          "className": "main-content-text",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "The Rider's Best Friend",
            "className": "main-tagline",
          },
          "ref": null,
          "rendered": "The Rider's Best Friend",
          "type": "h1",
        },
        "type": "div",
      },
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="main-content"
>
          <div
                    className="main-content-text"
          >
                    <h1
                              className="main-tagline"
                    >
                              The Rider's Best Friend
                    </h1>
          </div>
</div>,
        "className": "main-content-wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="main-content-text"
>
            <h1
                        className="main-tagline"
            >
                        The Rider's Best Friend
            </h1>
</div>,
          "className": "main-content",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="main-tagline"
>
              The Rider's Best Friend
</h1>,
            "className": "main-content-text",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "The Rider's Best Friend",
              "className": "main-tagline",
            },
            "ref": null,
            "rendered": "The Rider's Best Friend",
            "type": "h1",
          },
          "type": "div",
        },
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
